test/run

```diff
@@ -162,7 +167,11 @@ test_application() {
   # Wait for the container to write it's CID file
   wait_for_cid
 
-  test_scl_usage "varnishd -V" "varnish-4"
+  if CONTAINER_ARGS="--rm" container_run rpm -q scl-utils; then
+    test_scl_usage "varnishd -V" "varnish-4"
+  else
+    test_scl_usage "varnishd -V" "varnishd (varnish-"
```

Comment:
Would be great to test right `varnishd` versions for Fedora too...

You can use `VERSION` variable which is set to name of directory (~SW version).

`test_scl_usage "varnishd -V" "varnishd (varnish-$VERSION"`
IMHO this could be used for both - SCL and Fedora images...

Reply:
good suggestion, will do!